src/App.js

```diff
@@ -1,85 +1,261 @@
 import React from 'react';
+import uuid from 'react-uuid';
+import * as cn from 'classnames';
```

Comment:
```suggestion
import cn from 'classnames';
```
так должно тоже работать